multibot.py

```diff
@@ -51,37 +49,42 @@ def adjustmad(self, pairs, mad):
     def payload(self, pairs, mad):
 
         payload = {
-            "name": self.prefix + "_" + self.subprefix + "_" + self.suffix,
+            "name": self.attributes.get("prefix", "3CQSBOT", self.dca_conf) \
+                    + "_" + self.attributes.get("subprefix", "MULTI", self.dca_conf) \
+                    + "_" + self.attributes.get("suffix", "standard", self.dca_conf),
             "account_id": self.account_data["id"],
             "pairs": pairs,
             "max_active_deals": mad,
-            "base_order_volume": self.attributes.get("bo"),
-            "take_profit": self.attributes.get("tp"),
-            "safety_order_volume": self.attributes.get("so"),
-            "martingale_volume_coefficient": self.attributes.get("os"),
-            "martingale_step_coefficient": self.attributes.get("ss"),
-            "max_safety_orders": self.attributes.get("mstc"),
-            "safety_order_step_percentage": self.attributes.get("sos"),
+            "base_order_volume": self.attributes.get("bo", "", self.dca_conf),
+            "take_profit": self.attributes.get("tp", "", self.dca_conf),
+            "safety_order_volume": self.attributes.get("so", "", self.dca_conf),
+            "martingale_volume_coefficient": self.attributes.get("os", "", self.dca_conf),
+            "martingale_step_coefficient": self.attributes.get("ss", "", self.dca_conf),
+            "max_safety_orders": self.attributes.get("mstc", "", self.dca_conf),
+            "safety_order_step_percentage": self.attributes.get("sos", "", self.dca_conf),
             "take_profit_type": "total",
-            "active_safety_orders_count": self.attributes.get("max"),
-            "cooldown": self.attributes.get("cooldown", 0),
+            "active_safety_orders_count": self.attributes.get("max", "", self.dca_conf),
+            "cooldown": self.attributes.get("cooldown", 0, self.dca_conf),
             "strategy_list": self.strategy(),
-            "trailing_enabled": self.attributes.get("trailing", False),
-            "trailing_deviation": self.attributes.get("trailing_deviation", 0.2),
-            "allowed_deals_on_same_pair": self.attributes.get("sdsp"),
-            "min_volume_btc_24h": self.attributes.get("btc_min_vol", 0),
-            "disable_after_deals_count": self.attributes.get("deals_count", 0),
```

Comment:
why has this attribute been deleted?

Reply:
The line:
     "disable_after_deals_count": self.attributes.get("deals_count", 0),

Was not working, 3commas was reporting an error

Reply:
@itzwolf this was your change - is it working on your side? @M1cha3lV which error do you get?

Reply:
Error of 3commas: disable after deals count has to be greater >= 1

Reply:
Using the website 3C will report greater than 1 required however in my testing to set it to a specific figure (20) and then unset it to nothing (0), worked with no obvious errors in the log shown. Refreshing the bot details page showed the setting would be removed.

Reply:
Additionally, a value is required else there is an odd situation how the bot reacts and the script repeatedly calls the /symrank command. To unset the value there has to a something given and 1 would never turn it off so I believe via API it is by design to allow a 0, where the website you just remove the value altogether and save.

Reply:
It seems to be a 3commas problem, because if this field is initially set to 0, then this error pops up.
![image](https://user-images.githubusercontent.com/100045502/163730080-bbb9c71e-56bf-486a-8aff-5c5654671575.png)
When you start this option and you leave it blank like this "", 3commas reports an error

Reply:
"disable_after_deals_count": self.attributes.get("deals_count", ""),

Reply:
Correct on the website that is true. Was this a new bot created or modifying an existing?

I just tested on an existing bot. Modifying the setting to 20 and then to unset using 0 works. I have not tried doing a "" in the code... I can give that a go. Just with no value there is definitely an issue.

Reply:
Just tried it. "disable_after_deals_count": self.attributes.get("deals_count", "")," results in the oddity where the bot will restart after initially running so you get multiple /symrank commands being sent to the TG chat randomly. Using "disable_after_deals_count": self.attributes.get("deals_count", 0)," results in no errors and removed the configured value.

Reply:
When creating a new bot, this option can only pass the 3commas API when a value >0 is given. Leaving it blank with "" does not work. Perhaps it works with a space given: 
"disable_after_deals_count": self.attributes.get("deals_count", " "),

Reply:
I see that now... So modification it works, new creation, it fails.

Reply:
Asking the 3commas API team for this problem. At the moment the only solution is to remove the key. Empty value for this parameter is not accepted.

The additional code lines after payload removes the empty key in "disable_after_deals_count". 

        if payload["disable_after_deals_count"]==0:
            payload.pop("disable_after_deals_count")